### PR TITLE
Tweak result logging for native function calls

### PR DESCRIPTION
**Pull request by platfowner**
Summary:
- Tweak result logging for native function calls
- Fix checkinPayload object path bug 
- Move signTx() to ChainUtil
- Add sendTransferTx.js utility

#254

### CODE REVIEW

File: db/functions.js [hunk 1]
```diff
@@ -612,7 +625,7 @@ class Functions {
     const checkinId = context.params.checkin_id;
     const valuePath = context.valuePath;
     const checkinPayload = this.db.getValue(this.getCheckinPayloadPathFromValuePath(valuePath));
-    const checkinAmount = _.get(checkinPayload, 'transaction.tx_body.operation.value', 0);
+    const checkinAmount = _.get(checkinPayload, 'tx_body.operation.value', 0);
```
Comment (platfowner) [inline db/functions.js:628]: Object path fixed.

**Review by liayoo — APPROVED** [review]
LGTM, thanks!

**Review by minsulee2 — APPROVED** [review]
LGTM!

**Comment by platfowner** [comment]
Thanks for the fast review. Now merging..

**Pull request merged**